api/podcast.go

```diff
@@ -263,3 +264,185 @@ func FeedPodcast(c *gin.Context) {
 	payload := []byte(xml.Header + string(feedXml))
 	c.Data(http.StatusOK, "application/xml; charset=utf-8", []byte(payload))
 }
+
+func FeedCollections(c *gin.Context) {
+	var config feedConfig
+	(&config).getConfig(c)
+
+	var author = map[string]string{
```

Comment:
Is this really needed ?
Why not use person_i18n ?
If we really need this so please extract into some package variable and not inside function call. This data is fairly static.

Reply:
Language codes should be with [our consts](https://github.com/Bnei-Baruch/archive-backend/blob/master/consts/consts.go#L85). 
You could map user input from old site style with [LANG2CODE](https://github.com/Bnei-Baruch/archive-backend/blob/master/consts/consts.go#L284)